src/DDTrace/Integrations/Lumen/V5/LumenIntegrationLoader.php

```diff
@@ -21,14 +21,18 @@ public function load()
         $span = GlobalTracer::get()->getRootScope()->getSpan();
         $span->overwriteOperationName('lumen.request');
         $span->setTag(Tag::SERVICE_NAME, $this->getAppName());
+        $span->setTag(Tag::COMPONENT, 'lumen');
         $span->setIntegration(LumenIntegration::getInstance());
         $span->setTraceAnalyticsCandidate();
 
         // prepareRequest() was added in Lumen 5.2
         // https://github.com/laravel/lumen-framework/blob/5.2/src/Application.php#L440
         dd_trace('Laravel\Lumen\Application', 'prepareRequest', function (Request $request) use ($span) {
+            $method = $request->getMethod();
+            $operationName = $method . " " . $request->getPathInfo();
```

Comment:
In general I think we've avoiding having http methods in the operation name because the `http.method` tag is already utilized.  Also, it'd be good to ensure that the used path info is of reasonable cardinality (tested w/ uuids and query parameters).